app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/ids/DraftReturnId.scala

```diff
@@ -12,17 +12,14 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *@
+ */
 
-@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.ids
 
-@this(
-    mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
-    pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading
-)
+import play.api.libs.json.Json
 
-@()(implicit request: Request[_], messages: Messages, appConfig: ViewConfig)
+final case class DraftReturnId(value: String) extends AnyVal
```

Comment:
since we're not using this for the actual` DraftReturn` case class, could we stick this in the `dms` package under `models` since this is the only place it's going to be used?

Reply:
Yes this was problematic. I actually did want to change the DraftReturn definition to use this but didn't have time to make that change.

Reply:
It's really for the Upscan stuff rather than the DMS stuff. Address it after 6th April